### DISCUSSION TITLE
Display multi-value in details list layout?

### DISCUSSION THREAD

**Discussion by brianpmccullough** [microsoft-search/pnp-modern-search#2756] [Q&A]
Hello, I am tinkering with Microsoft Search for external items.  One of the items is multi-value.  How do I display the values (say the first 5 values) from a multi-value field ?  I am using the details list layout at the moment, but in general how do I access all values, and then are there utility function that allows me to get them as a delimited string?  TIA!  cc/ @wobba

**Comment by wobba** [comment]
They are likely coming back as a long string with some separator. Then you need to split the string. Lot's of functions you can use from here: https://github.com/helpers/handlebars-helpers#string

**Comment by zkent** [comment]
wobba is correct about the helpers.  I have successfully used `split` to split a string into an array using a delimiter such as `|`.  There seems to be an issue splitting along a line break though.

**Reply by wobba** [reply]
Maybe we need to add a helper which support regex splits. Seems the built in only support strings.

**Comment by kasperbolarsen** [comment]
@brianpmccullough  Did you found a solution to this question?
